eth2-ssv/beacon/beacon.go

```diff
@@ -0,0 +1,74 @@
+package beacon
+
+import (
+	"context"
+	"github.com/bloxapp/ssv/slotqueue"
+	"github.com/herumi/bls-eth-go-binary/bls"
```

Comment:
Any plans to switch to blst? You can use prysmaticlabs/prysm/shared/bls I believe